cmd/buildah/umount.go

```diff
@@ -20,8 +20,8 @@ var (
 	umountCommand = cli.Command{
 		Name:                   "umount",
 		Aliases:                []string{"unmount"},
-		Usage:                  "Unmounts the root file system on the specified working containers",
-		Description:            "Unmounts the root file system on the specified working containers",
+		Usage:                  "Unmount the root file system on specified working containers",
```

Comment:
I think "the' should remain.

Reply:
Changed.  That "on" should have been an "of", though, so changed that, too.